packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2847,11 +2849,22 @@ export default class Meeting extends StatelessWebexPlugin {
       })
         .then((peerConnection) => {
           LoggerProxy.logger.info(`${LOG_HEADER} PeerConnection Received from attachMedia ${peerConnection}`);
+
+          // Check if Peer Connection is STABLE after 30 seconds
+          // If NOT STABLE then BAIL and end meeting
+          setTimeout(() => {
+            if (peerConnection.iceConnectionState !== ICE_STATE.CONNECTED) {
```

Comment:
It's possible that the ice state could go to connected immediately, but then have an issue putting it not in a connected state at exactly 30 seconds. Should we put logic in for when the peer connection goes to connected to cancel this timeout? @arun3528 thoughts?

Reply:
Ice connection state change handler is [here](https://github.com/webex/webex-js-sdk/blob/6465ed41306a6a3db97a3cbdca7d081af097f3e0/packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js#L5740). It feels like this could go there.

Reply:
+1 we can bail out once it goes to connected state , pc can go to failed and disconnect state with in 30 sec of connection if the network does down or VPN changes etc